blocks/rich-text/index.js

```diff
@@ -558,6 +560,21 @@ export class RichText extends Component {
 		if ( keyCode === BACKSPACE ) {
 			this.onChange();
 		}
+
+		this.scrollToCaret();
+	}
+
+	scrollToCaret() {
+		const caretRect = this.getEditorSelectionRect();
+		const caretHeight = caretRect.y;
+		if ( caretHeight !== this.caretHeight ) {
+			const toolbarOffset = 100;
```

Comment:
Which toolbar does this refer to? And should the RichText component have built-in knowledge of said toolbar's height? When we redesign the toolbar, who will know to update this value accordingly?

Reply:
> And should the RichText component have built-in knowledge of said toolbar's height?

It shouldn't. :)

> Which toolbar does this refer to?

I grabbed the name from a comment in the parent PR, but I actually think of it independently of any toolbars. It's more of a "grace offset" so that we don't scroll to place the caret at the very top of the viewport: by leaving some empty space, we can incidentally accommodate toolbars, but it's more about scrolling to position the caret in a more central place, and perhaps revealing the end of the previous block to provide context.

> When we redesign the toolbar, who will know to update this value accordingly?

From my previous reasoning, there's no imperative to synchronize the values. That said, it is a magic constant, and is disconnected from the toolbar. Other ways of addressing include computing the offset dynamically, e.g. `1/3 · viewportHeight`.

Until further feedback, I'll rename and comment to clarify the intent.